service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/AzureStorageAccessService.java

```diff
@@ -96,6 +103,44 @@ private BlobContainerSasPermission getSasTokenPermissions(
     return BlobContainerSasPermission.parse(effectivePermissions);
   }
 
+  /**
+   * Creates a SAS token with a default start and expiry (drawn from app-level config)
+   *
+   * @param workspaceUuid id of the workspace that owns the container resource
+   * @param storageContainerResource storage container object we want to mint a SAS for
+   * @param storageAccountResource storage account which owns the storage container object
+   * @param userRequest The authenticated user's request
+   * @param sasIpRange (optional) IP address or range of IPs from which the Azure APIs will accept
+   *     requests for the token being minted.
+   * @return A bundle of 1) a full Azure SAS URL, including the storage account hostname and 2) the
+   *     token query param fragment
+   */
+  public AzureSasBundle createAzureStorageContainerSasToken(
```

Comment:
I've added a convenience overload for creating a sas token with the default start + expiry times to avoid having to wire in the azure configuration to the blob copying code and passing in the config'ed values there.